packages/frontend/src/components/otherSites/OtherSites.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
```

Comment:
rename directory from otherSites to other-sites as this is the convention we use